packages/yourloops/components/alarm/alarm-content-configuration.util.ts

```diff
@@ -71,4 +71,16 @@ export const buildBgValues = (bgUnit: UnitsType): BgValues => {
   }
   return { ...DEFAULT_BG_VALUES }
 }
-export const REGEX_VALUE_BG = /^(\d)*(.)?([0-9]{1})?$/
+
+const convertAndFormatBgValue = (value: number, currentUnit: UnitsType): number => {
+  const newUnit = currentUnit === UnitsType.MGDL ? UnitsType.MMOLL : UnitsType.MGDL
+  const formattedValueString = formatBgValue(convertBG(value, currentUnit), newUnit)
+
+  return newUnit === UnitsType.MGDL ? parseInt(formattedValueString) : parseFloat(formattedValueString)
+}
+
+export const getConvertedValue = (value: number, currentUnit: UnitsType, requiredUnit: UnitsType): number => {
```

Comment:
Question : Couldn't we put an interface for the parameters of the function ?
 ` interface ConvertedValueParameters {
    value: number, 
    currentUnit: UnitsType, 
    requiredUnit: UnitsType
}`

Reply:
it's a good question =), I would say to not have too many parameters in the function but maybe it's not useful